CombiningGame/Generators/TwoRandomOneToTenNumbersGenerator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Windows.Controls;
+
+namespace CombiningGame.Generators
+{
+    class TwoRandomOneToTenNumbersGenerator
+    {
+        public int RightAnswer;
+        public int firstNumber;
+        public int secondNumber;
+
+        public void GenerateRandomNumbers()
+        {
+            var random = new Random();
+            firstNumber = random.Next(1, 10);
+            secondNumber = random.Next(1, 10);
+            RightAnswer = firstNumber * secondNumber;
```

Comment:
Use get in RightAnswer. Don't init when first or second number is changed.

Reply:
?